library/n2_protocol.hpp

```diff
@@ -103,4 +103,14 @@ struct lookup_response : public n2_body {
 	                std::vector<unsigned char> data_checksum);
 };
 
+struct remove_request : n2_body {
+	uint64_t ioflags;
```

Comment:
Please support this request here: https://github.com/interiorem/elliptics/blob/4fc279c5a90efc4e2b2e68bd4c2d44448f28ea1f/cache/cache.cpp#L705